packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFGeneratedWebKitApis.h

```diff
@@ -700,4 +714,39 @@ NSObject<FlutterMessageCodec> *FWFWKHttpCookieStoreHostApiGetCodec(void);
 extern void FWFWKHttpCookieStoreHostApiSetup(id<FlutterBinaryMessenger> binaryMessenger,
                                              NSObject<FWFWKHttpCookieStoreHostApi> *_Nullable api);
 
+/// The codec used by FWFNSUrlHostApi.
+NSObject<FlutterMessageCodec> *FWFNSUrlHostApiGetCodec(void);
+
+/// Host API for `NSUrl`.
```

Comment:
Just to preserve this for posterity on the PR that actually lands it, I'm not sure NSURL is actually something we should wrap: https://github.com/flutter/plugins/pull/7113#discussion_r1103133475

(But as I said there, it's fine to land this as-is, just something I want to have a record of when we're looking at the pros and cons of the direct-wrap approach.)

Reply:
I'll keep this in mind. My main rationale for it was that NSURL has some useful methods/constructors that are unique to iOS.